internal/compiler/wasm/opa/opa.go

```diff
@@ -1,51 +1,27 @@
-// Copyright 2018 The OPA Authors.  All rights reserved.
+// Copyright 2021 The OPA Authors.  All rights reserved.
 // Use of this source code is governed by an Apache2
 // license that can be found in the LICENSE file.
 
-// THIS FILE IS GENERATED. DO NOT EDIT.
-
 // Package opa contains bytecode for the OPA-WASM library.
 package opa
 
 import (
-	"bytes"
-	"compress/gzip"
-	"io/ioutil"
-	"sync"
+	_ "embed"
 )
 
-var (
-	bytesOnce        sync.Once
-	bs               []byte
-	callGraphCSVOnce sync.Once
-	callGraphCSV     []byte
-)
+//go:embed opa.wasm
+var wasmBase []byte
+
+//go:embed callgraph.csv
+var callGraphCSV []byte
```

Comment:
This is really cool! I remember playing around with a similar trick in Rust using macros a while back. :smiley:  It's neat that this is a standard, supported thing in Golang now.